### ISSUE TITLE
Will not render linux. White screen displayed

### ISSUE THREAD

**Issue by kd8bny** [r0x0r/pywebview#890]
### Specification
- pywebview version: 3.6.3
- operating system: Archlinux 
- web renderer: qt
- python 3.8 and 3.10

### Description  

When starting a window, only a white screen is rendered.
Works on my windows machine as is, no errors are displayed.
Im really sure how to debug next.
  
### Practicalities

- YES I am willing to work on this issue myself.

- NO I am prepared to support this issue financially.

```
import webview

_ip = 'localhost'
_port = '5010'
_app_dir = 'trackplotter'
_url = 'http://{}:{}/{}'.format(_ip, _port, _app_dir)

# Start the window
window = webview.create_window(
Constants.application_titles['fts']['long'], _url, confirm_close=True)

webview.start(debug=True)
```

**Comment by kd8bny** [comment]
ope, found it
`systemd-coredump[55768]: [LNK] Process 55762 (QtWebEngineProc) of user 1000 dumped core.`

**Comment by marcelodiaz558** [comment]
Hey! Are you using Ubuntu 22.04? My app was working perfectly on both Ubuntu 20.04 and 21.10 but once I updated to 22.04 yesterday this same white screen problem started.

I use Python 3.10 and Pywebview 3.4, I updated my Pywebview to 3.6.3 and the problem persists. My Qt version is 5.15.2

**Comment by kd8bny** [comment]
I was using arch
Try running it again and then run 'journalctl' if you have a qtweb seg fault then it is the same issue.

**Comment by hohaicongthuan** [comment]
Same problem here but I **fixed** the problem myself. I'm using **Kubuntu 22.04**, **pywebview  3.6.3**, **Python 3.10.4**.

Apparently, using pywebview with Qt gives me a white blank window and I cannot right click or do anything with it so I **switch to GTK** and it worked.

Here's what I did:
1. Create a new virtual environment and activate it.
2. Install pywebview using `pip install pywebview`.
3. Install PyGObject dependencies using `sudo apt install python3-gi python3-gi-cairo gir1.2-gtk-3.0 gir1.2-webkit2-4.0`.
4. Install PyGObject using `pip install PyGObject` (at this step I got errors and it tell me to install this `sudo apt install libgirepository1.0-dev` so I did).
5. Run your python file like normal.

Hope this helps
